tests/TestCase/I18n/DateTimeTest.php

```diff
@@ -507,7 +510,11 @@ public function testToString(): void
         $time = new DateTime('2014-04-20 22:10');
         DateTime::setDefaultLocale('fr-FR');
         DateTime::setToStringFormat(IntlDateFormatter::FULL);
-        $this->assertTimeFormat('dimanche 20 avril 2014 22:10:00 UTC', (string)$time);
+        $expected = 'dimanche 20 avril 2014 22:10:00 UTC';
+        if (PHP_OS_FAMILY === 'Windows') {
+            $expected = 'dimanche 20 avril 2014 22:10:00 temps universel coordonné';
```

Comment:
I wonder if all windows installs will have this value? :thinking: We could switch to a regex in the future if need be.

Reply:
Yeah this is a weird one, we can adjust as needed in future.